Macros/Apply AHS Formats.SAS

```diff
@@ -0,0 +1,191 @@
+/* Apply AHS Formats.SAS */
+* Based on Apply AHS 2013 Formats.SAS;
+* 02/02/2017 9:41 AM; 
+* Program written by Shawn Bucholtz (shawn.j.bucholtz@hud.gov);
+* Edits by Dav Vandenbroucke (david.a.vandenbroucke@hud.gov);
+/*
+     This program inputs the value labels csv file provided by HUD and applies formats
+     to the American Housing Survey Public Use Files.
+     
+     1. It writes the format catalog to a specified SAS library.
+     2. It reads the SAS public use files from a specified library and saves them to
+          the same library, OVERWRITING the original files and converting them to 
+          native encoding.
+     
+*/
+
+/* User Customization Section ----------------------------------------------------------*/
+
+/* Specify survey year */
+%LET Year = 2017;
+TITLE Apply AHS &Year. Formats;
+
+/* Specify the list of relational files.  Uncomment the one that is applicable and comment out the one that is not */
+%LET RelFileList = household /*Mortgage*/ person project; /* this is for 2015 */
+/*%LET RelFileList = newhouse homimp mortg omov owner person ratiov rmov topical;*/ /* this is for 2013 */
+
+/* Specify location of value labels csv file.  INCLUDE the trailing backslash */
+%Let InDir = L:\Libraries\RegHsg\Raw\AHS 2017 Value Labels Package\;
+
+%LET CSVFile = AHS &Year. Value Labels.CSV;   /* name of input CSV file*/ 
+
+/* Specify location of AHS Public Use files. Include trailing backslash. Process metro and national files seperately from different file locations*/
+LIBNAME AHS "L:\Libraries\RegHsg\Raw\AHS 2017 Metropolitan PUF v1.0 SAS" ;
+
+/* location where you want the format catlog to be stored */
+LIBNAME Cat  "L:\Libraries\RegHsg\Raw\AHS 2017 Metropolitan PUF v1.0 SAS\" ;
```

Comment:
this should be to data not raw - that's probably why the other program isn't picking up the formats.